pages/linux/pro.md

```diff
@@ -0,0 +1,24 @@
+# pro
+
+> Manage Ubuntu Pro services.
+> More information: <https://ubuntu.com/pro>.
+
```

Comment:
```suggestion

- Connect your system to the Ubuntu Pro support contract:

`pro attach {{token}}`

```

Reply:
The command needs to be run as root and IMHO using the interactive mode is more user friendly than passing a token:
```suggestion

- Connect your system to the Ubuntu Pro support contract:

`sudo pro attach`

```